src/components/member-profile/index.js

```diff
@@ -11,6 +11,10 @@ import ShowSkills from '@components/member-card/show-skills';
 import { useForm } from 'react-hook-form';
 import { isEmail, isDecimal } from 'validator';
 import { KEY_ESC, KEY_TAB } from '@constants/AppConstants';
+import MemberTaskUpdate from '@components/member-task-update';
+import { userContext } from '@store/user/user-context';
```

Comment:
Maybe not relevant to your PR, but a question for future: Does the file need to be called `user-context`? It's already in the `user` folder.

Reply:
The user folder also has an index file that exports member context, so I guess it is good to have the file name as user-context.

Reply:
That PR doesn't contain those changes, but will refactor the code in a different PR as discussed